app.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra/base'
+require 'pg'
+require './lib/chitter.rb'
+
+class ChitterPost < Sinatra::Base
+
+  get '/' do
+    erb :index
+  end
+
+  get '/post' do
+    erb :post
+  end
+
+  post '/add_post' do
+    Chitter.create(message: params[:peep_txt])
+    redirect to('/live')
+  end
+
+  get '/live' do
+    @chitter = Chitter.all
+    erb :live
+  end
+
```

Comment:
Good 'clean' code in the controller file, which seems to adhere to SRP well with each get/post route having one responsibility.